sceptre imports causes plugin build failure on circleci

A few python modules are importing with a "." notation.  This seems to cause plugin builds to fail on CircleCi with an import error.  Note, I only see this failing on CircleCi, it seems to work when i run `make test` locally.

For example, in connection_manager.py there is this..
```
from .helpers import mask_key
from .exceptions import InvalidAWSCredentialsError, RetryLimitExceededError
```

Failure on CircleCi
https://circleci.com/gh/Sceptre/sceptre-ssm-resolver/16?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link
```
#!/bin/bash -eo pipefail
. venv/bin/activate
mkdir test-reports
make test
pytest --junitxml=test-reports/junit.xml
============================= test session starts ==============================
platform linux -- Python 3.6.8, pytest-3.2.5, py-1.8.0, pluggy-0.4.0
rootdir: /home/circleci/project, inifile:
collected 0 items / 1 errors                                                    

------ generated xml file: /home/circleci/project/test-reports/junit.xml -------
==================================== ERRORS ====================================
______________________ ERROR collecting tests/test_ssm.py ______________________
ImportError while importing test module '/home/circleci/project/tests/test_ssm.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
tests/test_ssm.py:8: in <module>
    from sceptre.connection_manager import ConnectionManager
venv/lib/python3.6/site-packages/sceptre/connection_manager.py:21: in <module>
    from .exceptions import InvalidAWSCredentialsError, RetryLimitExceededError
E   ImportError: cannot import name 'InvalidAWSCredentialsError'
!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!
=========================== 1 error in 0.32 seconds ============================
make: *** [Makefile:10: test] Error 2
Exited with code 2
```

--- a/sceptre/connection_manager.py
+++ b/sceptre/connection_manager.py
@@ -17,8 +17,8 @@ import boto3
 from os import environ
 from botocore.exceptions import ClientError
 
-from .helpers import mask_key
-from .exceptions import InvalidAWSCredentialsError, RetryLimitExceededError
+from sceptre.helpers import mask_key
+from sceptre.exceptions import InvalidAWSCredentialsError, RetryLimitExceededError
 
 
 def _retry_boto_call(func):

